logviewer/src/main/java/org/togetherjava/logwatcher/entities/LogDTO.java

```diff
@@ -0,0 +1,131 @@
+package org.togetherjava.logwatcher.entities;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Instant;
+
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class LogDTO {
```

Comment:
Personally, I do not like suffixing `DTO` or similar. If you can agree to that and its appicable, maybe rename all of them to just `Log`, `User`, ... (also alignment with main application module)

I do not have a strong opinion here though.